apps/deploy-web/env/.env

```diff
@@ -26,4 +25,7 @@ BASE_API_MAINNET_URL=$NEXT_PUBLIC_BASE_API_MAINNET_URL
 BASE_API_TESTNET_URL=$NEXT_PUBLIC_BASE_API_TESTNET_URL
 BASE_API_SANDBOX_URL=$NEXT_PUBLIC_BASE_API_SANDBOX_URL
 
-NEXT_PUBLIC_NODE_ENV=$NODE_ENV
\ No newline at end of file
+NEXT_PUBLIC_NODE_ENV=$NODE_ENV
+
+# TODO: replace once live and enabled
```

Comment:
reminder

Reply:
Yea, but this is not gonna be fixed on this pr